core/storage/user/gae_models.py

```diff
@@ -110,6 +110,22 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = ndb.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # Displayed name for users on android. They are editable unlike usernames.
+    name = ndb.StringProperty(default=None)
+    # A code associated with profiles to provide a PIN based authentication
+    # within the account on android.
+    pin = ndb.StringProperty(default=None)
+    # List of profile user ids associated with a full user. None for profiles.
```

Comment:
Write "None for profiles" as a fuller sentence -- "This is None if the user model corresponds to a ..."

Reply:
Done.